build.sh

```diff
@@ -270,11 +276,15 @@ yq . -o yaml ./config.toml | \
   yq e ".target.node = \"$target_node\"" -o json \
   > $target_dist_dir/config.json
 
-out_deb_file="$package_binary_name-$meta_version-$target.deb"
-out_bz2_file="$package_binary_name-$meta_version-$target.tar.bz2"
-out_pkg_file="$package_binary_name-$meta_version-$target-installer.pkg"
-out_dmg_file="$package_binary_name-$meta_version-$target.dmg"
-out_win_file="$package_binary_name-$meta_version-$target-installer.exe"
+variety_suffix=""
```

Comment:
What's that for? It doesn't seem to be used.

Reply:
Sorry, forgot to remove when changing the naming rule according to https://github.com/aclap-dev/vdhcoapp/pull/188#issuecomment-1857555562 and https://github.com/aclap-dev/vdhcoapp/pull/188#issuecomment-1857576272.

Now removed.